### ISSUE TITLE
[Accessibility]A11y_VSCode_Feedback_Screenreader:Voiceover does not read the text box name when focus goes to text box.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52208]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
ScreenReader,MAS1.3.1,VoiceOver,MACOS High Sierra 10.13.5 Beta (17F70a)
**Repro Steps:**
1. Launch VS Code.
2. Navigate to Status Bar and select "Tweet Feedback" button.
3. Check what Voiceover is reading.

Actual:
Voiceover does not read the Tell us why text box when focus goes to text box.

**Expected:**
 Voiceover should read the Tell us why text box when focus goes to text box.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
The keyboard user will not be able to access the control.
MAS Reference:
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

Does this issue occur when all extensions are disabled?: Yes
**Attachment For Reference**
[[Accessibility]A11y_VSCode_Feedback_Screenreader.pptx](https://github.com/Microsoft/vscode/files/2111760/Accessibility.A11y_VSCode_Feedback_Screenreader.pptx)

**Comment by bpasero** [comment]
It was reading "Comments" which I think is not very helpful, I changed it to: `Tell us your feedback`
